### DISCUSSION TITLE
It look like some issue with symlink

### DISCUSSION THREAD

**Discussion by BrandonWanHuanSheng** [microsoft/winget-pkgs#112463] [Q&A] [answered]
![Screenshot (259)](https://github.com/microsoft/winget-pkgs/assets/118059864/da50a564-b849-4c09-998d-327d2ab748b3)
It might execute failed for winget to add symlink. [Package Request: PDW](https://github.com/microsoft/winget-pkgs/issues/112214)

**Comment by Trenly** [comment, marked as answer]
Its the same issue with dll's not being available through symlink. If any files are directly referenced by an executable, then the symlink won't work for the executable
